src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/CreateSDORespondent1NotificationHandlerTest.java

```diff
@@ -50,13 +55,17 @@ public class CreateSDORespondent1NotificationHandlerTest extends BaseCallbackHan
     private OrganisationService organisationService;
     @Autowired
     private CreateSDORespondent1NotificationHandler handler;
+    private final String defendantEmail = "respondent@example.com";
```

Comment:
these should be private static final String DEFENDANT_EMAIL etc.

Reply:
done